src/axolotl/prompt_strategies/input_output.py

```diff
@@ -0,0 +1,45 @@
+"""Module for plain input/output prompt pairs"""
+
+from axolotl.prompt_tokenizers import PromptTokenizingStrategy
+
```

Comment:
```suggestion
from axolotl.prompters import IGNORE_TOKEN_ID
```
I think we want to bring this value in just like the other strategies?